README.md

```diff
@@ -192,7 +192,7 @@ As Helm does not support deleting or updating CRDs, they have to be deleted manu
 bash scripts/cleanup.sh
 ```
 
-*NOTE*: `cleanup.sh` will remove everything. Modify it to ignore dependencies that you might have installed separately.
+*NOTE*: `cleanup.sh` will remove everything. Modify it to ignore dependencies that you might have installed separately, see [Disabling dependencies](#disabling-dependencies).
```

Comment:
I was rather thinking of linking this from `#disabling-dependencies`, but I guess it's OK as-is.